randovania/generator/hint_distributor.py

```diff
@@ -37,7 +36,6 @@ def _not_empty(it: Iterator) -> bool:
 @dataclasses.dataclass(frozen=True)
 class PreFillParams:
     rng: Random
-    configuration: BaseConfiguration
```

Comment:
Any reason why this field is being removed?

Reply:
I noticed something wasn't used so I removed it. I can revert this change if desired.

Reply:
It's more that it was expected to be used, but didn't end up being the case. It's likely to be needed if any game want to configure the hints a bit more though.

Reply:
reverted